db/change_cache.go

```diff
@@ -519,7 +521,7 @@ func (c *changeCache) DocChanged(event sgbucket.FeedEvent) {
 		TimeSaved:    syncData.TimeSaved,
 		Channels:     syncData.Channels,
 	}
-	base.Infof(base.KeyDCP, "Received #%d after %3dms (%q / %q)", change.Sequence, int(feedLatency/time.Millisecond), base.UD(change.DocID), change.RevID)
```

Comment:
This might be out of scope, but one of the things we discussed at some point was logging this at `info` if the latency exceeds some threshold. Worth adding that now?

Worst case scenario is that you'd get noisy logs *if* your SG clocks are out of sync - which is probably something to be aware of anyway.

Reply:
++ On that last bit, if we're adding anything, might be nice to have a similar warning for negative values...

Reply:
@JFlath What kind of value were you thinking for the threshold?

Reply:
Fair ask. 

Stab in the dark: 1m maybe?

Reply:
👍